src/app/app-routing.module.ts

```diff
@@ -14,12 +14,13 @@ import {
 
 
 const routes: Routes = [
-  { path: '', redirectTo: 'splash', pathMatch: 'full' },
-  { path: 'splash', component: SplashComponent },
+
+
   {
     path: 'home',
     component: HomeComponent,
     children: [
+      { path: 'splash', component: SplashComponent,  outlet: 'sidenav'},
       { path: 'mayor', component: MayorComponent, outlet: 'sidenav' },
       { path: 'city-attorney', component: CityAttorneyComponent, outlet: 'sidenav' },
       {
```

Comment:
Checked out your branch and seeing how to mesh the new layout together.  For this part of the app-routing module, we don't need the parent-child routing like you see in the object home path.  We can simply do just this:

`{ path: '', redirectTo: 'splash',pathMatch: 'full' },`
`{ path: 'splash', component: SplashComponent },`
`{ path: 'mayor', component: MayorComponent }`
`{ path: 'under-construction', component: UnderConstructionComponent },`

We can also omit the `outlet: 'sidenav'` part as well from each of these paths.

Reply:
@gSoD17 Pushed these changes. Check it out.